src/nbl/asset/utils/CHLSLCompiler.cpp

```diff
@@ -25,22 +25,190 @@
 #include <boost/wave/cpplexer/cpp_lex_token.hpp>
 #include <boost/wave/cpplexer/cpp_lex_iterator.hpp>
 
-#define TCPP_IMPLEMENTATION
-#include <tcpp/source/tcppLibrary.hpp>
-#undef TCPP_IMPLEMENTATION
+//#define TCPP_IMPLEMENTATION
+//#include <tcpp/source/tcppLibrary.hpp>
+//#undef TCPP_IMPLEMENTATION
 
 using namespace nbl;
 using namespace nbl::asset;
 using Microsoft::WRL::ComPtr;
 
 static constexpr const wchar_t* SHADER_MODEL_PROFILE = L"XX_6_7";
 
+
+
 namespace nbl::asset::hlsl::impl
```

Comment:
remove the `hlsl` namespace while you're at it, it looks weird, just keep the impl